certbot/bin/run_certbot.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
+# shellcheck disable=SC2039,SC2113
+
+set -e
+
+print_usage() {
+cat << EOF
+usage: ${0} options
+
+This script helps to update content views in Foreman.
+
+OPTIONS:
+    Parameters:
+    -d Domain for certificate.
+    -e Administrator email used when obtaining certificates.
+    -r Webroot location to place challenge auth files.
+    -A Additional arguments to pass to certbot command.
+    -D Enable debug mode for additional output.
+    -S Enable staging mode for test certificates.
+
+Version: ${VERSION}
+EOF
+}
+
+function check_hostname {
```

Comment:
Suggest a different name: `is_ipaddress` so the return values make sense. Right now it feels counterintuitive since the function is called `check_hostname` and returns `0` for IP